shared/package.json

```diff
@@ -37,6 +37,7 @@
     "caniuse-lite": "^1.0.30001081",
     "jsoneditor": "^8.6.1",
     "jwt-decode": "^3.1.2",
+    "monaco-editor": "^0.23.0",
```

Comment:
why do we use `monaco-editor` here and `monaco-editor/react` in main package.json? Those are 2 different libraries and all our dependencies in the code use `monaco-editor/react`.

Reply:
Additionally, we use `monaco-editor/react` not only in shared, but also in core-ui. It would make more sense to put it in the main package.json